Makefile.PL

```diff
@@ -464,11 +464,7 @@ my %o = ( 'NAME' => 'DBD::MariaDB',
 	  'OBJECT' => '$(O_FILES)',
 	  'LIBS' => (join ' ', @libdirs, $main_lib, @libs),
 	  @ldflags ? (dynamic_lib => { OTHERLDFLAGS => (join ' ', @ldflags) }) : (),
-	  'VERSION_FROM' => 'lib/DBD/MariaDB.pm'
-	);
-
-if (eval { ExtUtils::MakeMaker->VERSION(5.43) }) {
-  %o=(%o,
+	  'VERSION_FROM' => 'lib/DBD/MariaDB.pm',
```

Comment:
This breaks indentation if tab width != 2.

Reply:
All lines before `'VERSION_FROM'` have same style of indentation. (Ehm... it is a style? :D)

The only change which I done for this `'VERSION_FROM'` line was adding a trailing comma. No other whitespace change.

Reply:
Yes, but the following lines have a different "style".

Reply:
I see... but I'm not sure what I can do with it. Some lines use only spaces, some use TAB+2spaces and some only TABs.

Reply:
🤷🏽 select one style (probably the most common one) and apply it to the whole block? Maybe in a separate commit so it's clear nothing else changes.

Reply:
Ok, I tried something in new commit.